pkgs/development/python-modules/twisted/default.nix

```diff
@@ -56,5 +56,6 @@ buildPythonPackage rec {
     '';
     license = licenses.mit;
     maintainers = [ ];
+    knownVulnerabilities = [ "CVE-2020-10108" "CVE-2020-10109" ];
```

Comment:
hmm, won't this mean on an upgrade any package that depends on this version of twisted will fail to evaluate?

Reply:
Yes.

And this is basically the main reason I asked for input before merging/pushing: the thing is, to fix this package (by bumping it), it's necessary to fix several dependencies (and as you can see this affects even pytest in the end) and I'm not even sure if there's an even bigger fallout after this. So in the end, I'm absolutely not sure what's the best (acceptable) solution here :)

Reply:
This is how fedora patched 10109 https://src.fedoraproject.org/rpms/python-twisted/c/9248b58fc9b22a159f50759ab4959619dfa04a04?branch=f31 you would have to do some sort of upgrade still. I don't think we can mark it as knownVulnerabilities

Reply:
This is why it is so important we update all our core packages before doing a release. Apparently the twisted version here was already quite dated, so patching it is tough.

Reply:
Applied the suggestion of @worldofpeace now.